extensions/amp-wistia-player/validator-amp-wistia-player.protoascii

```diff
@@ -33,6 +33,13 @@ tags: {  # <amp-wistia-player>
     mandatory: true
     value_regex: "[0-9a-zA-Z]+"
   }
+  attrs: {
+    # Does nothing, as the player iframe implements this feature internally
```

Comment:
I'm not sure these comments are necessary but you're welcome to keep them.

I did look at `amp-vimeo` and `amp-reach-player`. Neither of those have this attribute either.

Reply:
`amp-video` does. The attribute is being rolled out as compatibility is audited, etc.